fm4tlp/test.py

```diff
@@ -195,7 +201,7 @@ def main(_):
   device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
 
   # Start...
-  unused_start_overall = timeit.default_timer()
```

Comment:
Please keep `unused_` before variables that are not used after their declaration. If you make a change that uses them, they can be renamed.